DuckDuckGo/CrashReports/Model/CrashReporter.swift

```diff
@@ -24,7 +24,8 @@ import PixelKit
 final class CrashReporter {
 
     private let reader = CrashReportReader()
-    private lazy var sender = CrashReportSender(platform: .macOS)
+    private lazy var sender = CrashReportSender(platform: .macOS, pixelEvents: CrashReportSender.pixelEvents)
```

Comment:
These failures are presumably because the updates to BSK aren’t available yet, does that sound right?  I’d expect to see the same thing in iOS, but those github actions seem to have stalled without reporting back to my PR :-/